Project EFT/Views/Shared/EditInfo.cshtml

```diff
@@ -0,0 +1,108 @@
+</form>
+<form id="passwordForm" action="~/EditInfo/editPassword" method="post">
+    <input type="hidden" id="password" name="password" />
+</form>
+    Enter New Password:
+<br>
+<input type="password" id="passwordEntry" name="passwordEntry" />
+<br>
+<div class="error" id="password-error"></div>
+    Confirm New Password:
+<br>
+<input type="password" id="confirmPassword" name="confirmPassword" />
+<br>
+<div class="error" id="confirm-password-error"></div>
+<input type="button" value="Update Password" onclick="verifyPassword()">
+<div class="error">@passwordError</div>
+<div class="success">@passwordSuccess</div>
+
+<script>
+
+    function verifyUsername() {
+        let usernameElement = document.getElementById("username");
+        let username = usernameElement.value.trim();
+        let usernameGood = username.length > 0;
+        if (usernameGood) {
+            usernameElement.value = username;
+            document.getElementById("username-error").innerHTML = "";
+            document.getElementById("usernameForm").submit();
+        } else {
+            document.getElementById("username-error").innerHTML = "Please enter a valid username.";
+        }
+    }
+
+    function verifyEmail() {
+        let emailElement = document.getElementById("email");
+        let email = emailElement.value.trim();
+        let emailGood = verifyInformation(email, TYPE_EMAIL);
+        if (emailGood) {
+            emailElement.value = email;
+            document.getElementById("email-error").innerHTML = "";
+            document.getElementById("emailForm").submit();
+        } else {
+            document.getElementById("email-error").innerHTML = "Please enter a valid email.";
+        }
+    }
+
+    function verifyPassword() {
+        let passwordEntryElement = document.getElementById("passwordEntry");
+        let confirmPasswordElement = document.getElementById("confirmPassword");
+        let passwordSubmissionElement = document.getElementById("password");
+        let password = passwordEntryElement.value.trim();
+        let confirmPassword = confirmPasswordElement.value.trim();
+        let passwordComplex = verifyInformation(password, TYPE_PASSWORD);
+        let passwordsMatch = password == confirmPassword;
+        if (passwordComplex && passwordsMatch) {
+            passwordSubmissionElement.value = md5(password);
+            document.getElementById("password-error").innerHTML = "";
+            document.getElementById("confirm-password-error").innerHTML = "";
+            document.getElementById("passwordForm").submit();
+        } else {
+            document.getElementById("password-error").innerHTML = passwordComplex ? "" : "Please enter a valid password. <br> A valid password is at least 8 characters, and includes at least one number, uppercase letter, and symbol.";
+            document.getElementById("confirm-password-error").innerHTML = passwordsMatch ? "" : "Passwords must match.";
+        }
+    }
+
+</script>
```

Comment:
Could this just be added to site.js?  Not sure we would need it on a page other than here, but it may just be best practice to keep all js functions in one place.